examples/prom-statsd-exporter/conf.yaml

```diff
@@ -67,6 +67,13 @@ mappings: # Requires statsd exporter >= v0.6.0 since it uses the "drop" action.
     labels:
       err_type: "$1"
 
+  - match: "ratelimit_server.total_requests"
+    name: "ratelimit_service_total_requests"
+    match_metric_type: counter
+  - match: "ratelimit_server.response_time"
+    name: "ratelimit_service_response_time_seconds"
```

Comment:
`s.responseTime.AddValue(float64(time.Since(start).Milliseconds()))` is this in milliseconds or seconds ?